GPU/Common/TextureCacheCommon.cpp

```diff
@@ -2397,6 +2398,18 @@ bool TextureCacheCommon::PrepareBuildTexture(BuildTexturePlan &plan, TexCacheEnt
 		// But, we still need to create the texture at a larger size.
 		plan.replaced->GetSize(0, plan.createW, plan.createH);
 	} else {
+		if (replacer_.Enabled() && !plan.replaceValid && plan.depth == 1) {
+			ReplacedTextureDecodeInfo replacedInfo;
+			// TODO: Do we handle the race where a replacement becomes valid AFTER this but before we save?
```

Comment:
No, not anymore.  It before checked `IsInvalid()` (which is, importantly, not the same as `!IsValid()` - when the state is yet indeterminate) before saving.  Because that was removed, it will presumably overwrite or resave things while loading.

-[Unknown]